connect/res/values/strings.xml

```diff
@@ -15,19 +15,19 @@
     <string name="subscribers_name_connect" translatable="false">Telenor</string>
     <string name="subscribers_name_dtacid" translatable="false">dtac</string>
 
-    <string name="com_telenor_connect_login_button_text" formatted="true">Sign in with %1$s</string>
+    <string name="com_telenor_connect_login_button_text" formatted="true">Continue with %1$s</string>
     <string name="com_telenor_connect_error_text1" formatted="true">There was an error communicating with %1$s.</string>
     <string name="com_telenor_connect_error_text2">Please check your network settings or try again later.</string>
     <string name="com_telenor_connect_try_again">TRY AGAIN</string>
     <string name="com_telenor_connect_network_settings"><u>Network settings</u></string>
     <string name="com_telenor_connect_show_details">Show details</string>
     <string name="com_telenor_connect_hide_details">Hide details</string>
-    <string name="com_telenor_about_description" formatted="true">Your %1$s is used to sign in to all apps and services.\u0020</string>
+    <string name="com_telenor_about_description" formatted="true">Your %1$s is used to login to all apps and services.\u0020</string>
```

Comment:
to log in ("log in" is a verb phrase, "login" is a noun)

Reply:
Fixed